packages/oui-guide-menu/README.md

```diff
@@ -0,0 +1,39 @@
+# oui-guide-menu
```

Comment:
(optional) I think we could remove the LESS documentation of this component. Since it has only the documentation of the button :)

Reply:
oki. Makes sens to me too